### PR TITLE
BETA: Register giocoliere.is-a.dev

### PR DESCRIPTION
**Pull request by giocoliere**
Added `blog.gioco.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Review by VaibhavSys — CHANGES_REQUESTED** [review]
_atproto is currently not supported.

File: domains/giocoliere.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "giocoliere",
+        "email": "simcrigjeki@gmail.com"
+    },
+    "record": {
+        "CNAME": "exquisite-paprenjak-917b5f.netlify.app"
```
Comment (CuteDog5695) [inline domains/giocoliere.json:7]: we dont support netlify

File: domains/_atproto.gioco.json
Comment (CuteDog5695) [inline domains/_atproto.gioco.json:1]: delete this file please since we dont support _atproto

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by omsenjalia — CHANGES_REQUESTED** [review]
cannot reach web

**Pull request closed without merging**